Codecademy/gamut
# feat(Flyout): Added optional background prop

### Overview
Allow gamut Flyout component to have changeable background color. 





### PR Checklist

- [x] Related to designs: https://www.figma.com/file/Km8Nrk054HnZpjHsaSopLp/Filters?node-id=200%3A6307&t=zWVOWjn0zfKnn4RW-0
- [x] Related to JIRA ticket: [DISC-1007]

#### Testing Instructions



1. Open storybook link
2. Navigate to Flyout section 
3. Scroll down to list of props and see new `bg` prop 
4. Change that value to some color 
5. Click "Tell me more" above and see this color in the flyout 

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/33936)  | [Monolith Env](https://pr-33936-monolith.dev-eks.codecademy.com/) |
| Monorepo       | [Portal PR](https://github.com/codecademy-engineering/mono/pull/843)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-843)   |




[DISC-1007]: https://codecademy.atlassian.net/browse/DISC-1007?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
